### PR TITLE
Ft add clique setup

### PR DESCRIPTION
**Pull request by ppoliani**
Similar to the IBFT 2.0 network but using a NodeJS script to generate the config files for Clique.

### CODE REVIEW

File: kubectl/private-network-clique/configmap/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        "chainId": 1729,
+        "constantinoplefixblock": 0,
+        "clique":{
+          "blockperiodseconds":5,
+          "epochlength":30000
+        }
+      },
+      "coinbase":"0x0000000000000000000000000000000000000000",
+      "difficulty":"0x1",
+      "extraData":"0x0000000000000000000000000000000000000000000000000000000000000000<signers>0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
```
Comment (joshuafernandes) [inline kubectl/private-network-clique/configmap/configmap.yaml]: Hi @ppoliani could you update this to contain the addresses of the signers please? Idea being we'd like to get people going with the config in this template to see it working and we mention in the readme that they have to make their own keys, config etc for production.

File: kubectl/private-network-clique/README.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+mv ./cliqueSetup/networkFiles/genesis.json ./cliqueSetup/
+```
+
+Update the secrets/validator-key-secret.yaml with the private keys. The private keys are put into secrets and the public keys go into a configmap that other nodes use to create the enode address
+Update the configmap/configmap.yaml with the public keys
+**Note:** Please remove the '0x' prefix of the public keys
+
+#### 2. Genesis.json
+Copy the genesis.json file and copy its contents into the configmap/configmap as shown
+
```
Comment (joshuafernandes) [inline kubectl/private-network-clique/README.md]: Also worth adding something here with something along the lines of `Update the extra data with addresses that match you validators by concatenating them`

Comment (ppoliani) [reply kubectl/private-network-clique/README.md]: @joshuafernandes  `besu-clique-config` will take care of replacing creating the correct `extraData` in the generated genesis file. Basically, there is no need to add it manually.

Comment (joshuafernandes) [reply kubectl/private-network-clique/README.md]: :+1:

**Pull request merged**